run_dir/design/deliveries.html

```diff
@@ -23,44 +23,36 @@
 
 <div id="page_content" class="deliveries-page">
   <div class="row">
```

Comment:
I know that this isn't in this PR, but you should only have `.row` divs when the immediate children are columns. If you remove this `<div>` (and it's closing tag) then the page will have proper spacing at the edge.